test/python/circuit/test_circuit_qasm.py

```diff
@@ -176,8 +175,7 @@ def test_circuit_qasm_pi(self):
         """Test circuit qasm() method with pi params.
         """
         circuit = QuantumCircuit(2)
-        circuit.append(random_unitary(4, seed=1234), [0, 1])
-        circuit = circuit.decompose()
+        circuit.cz(0, 1)
```

Comment:
Was this change made because the random unitary decomposition to qasm and back would lose the global phase and the equality would differ now? FWIW I agree using a random unitary here seems unnecessary since it's testing the pi serialization for qasm, I'm just curious.

Reply:
That's right, the unitary ends up generating a non-zero global phase that's lost in the back and forth to QASM.